COPYRIGHT-NOTICES.md

```diff
@@ -0,0 +1,138 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+
+# Copyrights and OpenJS Foundation projects
+
+## Ownership of Copyrights in OpenJS Foundation Contributions
+
+When source code, documentation and other content is contributed to an Open JS Foundation
+project, the copyrights in those contributions remain owned by the original
+copyright holders.
+
+The copyrights are not _assigned_ to OpenJS Foundation. Instead, they are _licensed_ for
+distribution as part of the project. Whether a project uses the DCO or a CLA,
+the original copyright holders retain their copyrights.
+
+## Copyright Notices
+
+OpenJS Foundation does not require or recommend that every contributor include their
+copyright notice in contributed files. [See below for more details on why
```

Comment:
this seems to talk about "no notice" versus "a notice in every file", but doesn't seem to mention the far more reasonable option of "copyrights only go in the license file".

Reply:
With more organizations doing automatic license scanning, an unmodified, standard license file tends to be the least error-prone.  That said, projects can choose when and in which files to include these additional copyright notice lines, and this guidance applies whether they choose to put these additional lines in individual files, a single LICENSE file, or no files at all.

Reply:
The copyright section generally is a bit fuzzier for this reason; but i'm mainly concerned about any advice that suggests that it's a good idea to put boilerplate metadata in every file - that's what version control is for :-)

Reply:
Agreed, yes.  It gets out of date easily and has no guarantee of correctness.  That's one of the reasons this recommends against doing that.  But if you feel you need to, it provides guidance on how to do it properly.

Reply:
Suggested a session on these IP-related issues at the collab summit next week [here](https://docs.google.com/document/d/1hdyWxXjCXUe1iEUqaQE8jvt38N6ckQomxTCtgkdisnU/edit). Additional subtopics welcomed.

Reply:
Unfortunately I can't make it to the collab summit, but will be around all day Wednesday to discuss. I'd be happy to take as much time is needed for this. 

Please note though that I'm only the messenger and while I'm familiar with the docs and can ask questions of Foundation counsel, I don't approve changes. :-) Per the organizational bylaws, the IP policy is set by the board of directors, although I'm happy to help clarify where I can.

Reply:
Could point, @brianwarner. See my comment about this here: https://github.com/openjs-foundation/cross-project-council/issues/390#issuecomment-561740114. I feel like a joint session between the CPC and Board would be incredibly useful (if that's not already planned).

Reply:
I think one of the key things is capturing the feedback from the projects to help both legal and the board understand the concerns.  I see that as an ongoing process we need to make sure happens.  In this specific instance, while the documents may be new, they to a large extent are based on what was in place in the JS and Node.js foundations so not necessarily "new" in terms of content.  

As new projects join (like AMP) we will likely have new/different concerns and we need to get the process in place capture those and figure out how those got into updates of the existing docs like the IP policy.  @brianwarner I think you in addition to the CPC board reps (myself included) need to be part of the return feedback loop as opposed to just being "messengers".

Depending on the progress on capturing/documenting concerns/recommendations at the Collab summit next week we should define next steps.

In terms of documenting current guidance based on the approved Policy in the CPC repo, I'm wondering if that can be decoupled from desired updates?  My perspective is that as long as the doc is accurate based on the current IP policy it's better to have it documented versus not.